aws/resource_aws_macie2_member_test.go

```diff
@@ -0,0 +1,183 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/macie2"
+	"github.com/hashicorp/aws-sdk-go-base/tfawserr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func testAccAwsMacie2Member_basic(t *testing.T) {
+	var macie2Output macie2.GetMemberOutput
+	resourceName := "aws_macie2_member.test"
+	accountID := "520433213222"
+	email := "test@test.com"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckAwsMacie2MemberDestroy,
+		ErrorCheck:        testAccErrorCheck(t, macie2.EndpointsID),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAwsMacieMemberConfigBasic(accountID, email),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsMacie2MemberExists(resourceName, &macie2Output),
+					resource.TestCheckResourceAttr(resourceName, "relationship_status", macie2.RelationshipStatusCreated),
+					testAccCheckResourceAttrRfc3339(resourceName, "invited_at"),
+					testAccCheckResourceAttrRfc3339(resourceName, "updated_at"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccAwsMacie2Member_disappears(t *testing.T) {
+	var macie2Output macie2.GetMemberOutput
+	resourceName := "aws_macie2_member.test"
+	accountID := "520433213222"
+	email := "test@test.com"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckAwsMacie2MemberDestroy,
+		ErrorCheck:        testAccErrorCheck(t, macie2.EndpointsID),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAwsMacieMemberConfigBasic(accountID, email),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsMacie2MemberExists(resourceName, &macie2Output),
+					testAccCheckResourceDisappears(testAccProvider, resourceAwsMacie2Member(), resourceName),
+				),
+				ExpectNonEmptyPlan: true,
+			},
+		},
+	})
+}
+
```

Comment:
The test `testAccAwsMacie2InvitationAccepter_memberStatus` should actually be here, since it's the `status` on the `aws_macie2_member` that's changing.